.travis.yml

```diff
@@ -165,7 +165,8 @@ script:
   - python -m pytest -v --durations=10 test/tensorflow_test.py
   - python -m pytest -v --durations=10 test/failure_test.py
   - python -m pytest -v --durations=10 test/microbenchmarks.py
-  - python -m pytest -v --durations=10 test/stress_tests.py
+  - python -m pytest -v --durations=10 -s test/stress_tests.py::test_multiple_recursive
+  - python -m pytest -v --durations=10 test/stress_tests.py -k "not test_multiple_recursive"
```

Comment:
To clarify, do you think that the *only* reason for the test failure is that the test takes longer than 10 minutes and so Travis times out? That doesn't seem to fully explain it, right?